sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Azure.Monitor.OpenTelemetry.Exporter.csproj

```diff
@@ -3,7 +3,7 @@
     <Description>An OpenTelemetry .NET exporter that exports to Azure Monitor</Description>
     <AssemblyTitle>AzureMonitor OpenTelemetry Exporter</AssemblyTitle>
     <Version>1.0.0-beta.6</Version>
-    <PackageTags>Azure Monitor OpenTelemetry Exporter</PackageTags>
+    <PackageTags>Azure Monitor OpenTelemetry Exporter ApplicationInsights</PackageTags>
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/bf95a54901e08f1e057f1ba84200fb68eab041be/build/Common.prod.props#L56 No need of separators?

Reply:
It appears that it can be either a space or semi-colon delimeter. :)
See also: https://github.com/dotnet/docs/issues/13553